sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/azure-security-keyvault-jca-user-agent-value-prefix.txt

```diff
@@ -0,0 +1 @@
+az-sp-kv-ct/
```

Comment:
why is this called ct?

Reply:
It's 'ct' in legacy version [3.14,0](https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot-starter-keyvault-certificates_3.14.0/sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/resources/azure-security-keyvault-jca-user-agent-value-prefix.txt), and it's also used for [starter spring-cloud-azure-starter-keyvault-certificates](https://github.com/Azure/azure-sdk-for-java/blob/ed60f735647117b27d8ee1080fdd909303ffd654/sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java#L52), how about 'jca'?

Reply:
Let's call it `jca`, and add a ut for it.